.github/CONTRIBUTING.md

```diff
@@ -425,27 +475,27 @@ Math operators like +, -, /, *, etc are up in the air, just choose which version
 
 #### Use
 * Bitwise AND - '&'
-	* Should be written as ```bitfield & bitflag``` NEVER ```bitflag & bitfield```, both are valid, but the latter is confusing and nonstandard.
+	* Should be written as `variable & CONSTANT` NEVER `CONSTANT & variable`. Both are valid, but the latter is confusing and nonstandard.
```

Comment:
I'm not sure this is a good change, although field and flag aren't really explained here, maybe they should be

Reply:
which of "bitfield" and "bitflag" is the variable and which is the constant is not obvious even to me; I am only able to guess what the author meant based on what they said was correct

Reply:
That's true. Might be worth integrating this with the tg wiki's guide to bitflags from the yee olden days, but that's something to do later